product_form_sale_link/views/sale_order_line.xml

```diff
@@ -2,13 +2,32 @@
 <!-- Copyright 2019 ACSONE SA/NV
      License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
 <odoo>
+    <record id="sale_order_line_tree" model="ir.ui.view">
```

Comment:
The problem with this is that if one want to change the tree view fields, you must depends on this module.